core/session.ts

```diff
@@ -11,6 +11,11 @@ import { dataform } from "df/protos/ts";
 import { util } from "protobufjs";
 import { default as TarjanGraphConstructor, Graph as TarjanGraph } from "tarjan-graph";
 
+const DEFAULT_CONFIG = {
```

Comment:
what's this needed for? would rather not have it if at all possible, just to prevent drift from the default `dataform.json`

Reply:
Just preserving existing behaviour, refactoring code from below. I think this is mainly used in tests